package.json

```diff
@@ -15,50 +15,50 @@
     "@allmaps/iiif-parser": "1.0.0-beta.42",
     "@annotorious/annotorious": "^3.3.6",
     "@annotorious/react": "^3.3.6",
-    "@astrojs/netlify": "^6.3.2",
+    "@astrojs/netlify": "^6.3.3",
     "@astrojs/node": "^9.2.1",
     "@astrojs/react": "^4.2.7",
     "@dnd-kit/core": "^6.3.1",
     "@dnd-kit/sortable": "^10.0.0",
     "@dnd-kit/utilities": "^3.2.2",
-    "@phosphor-icons/react": "^2.1.7",
-    "@radix-ui/react-accordion": "^1.2.8",
-    "@radix-ui/react-alert-dialog": "^1.1.11",
-    "@radix-ui/react-avatar": "^1.1.7",
-    "@radix-ui/react-checkbox": "^1.2.3",
-    "@radix-ui/react-dialog": "^1.1.11",
-    "@radix-ui/react-dropdown-menu": "^2.1.12",
-    "@radix-ui/react-label": "^2.1.4",
-    "@radix-ui/react-popover": "^1.1.11",
-    "@radix-ui/react-progress": "^1.1.4",
-    "@radix-ui/react-radio-group": "^1.3.4",
-    "@radix-ui/react-scroll-area": "^1.2.6",
-    "@radix-ui/react-select": "^2.2.2",
-    "@radix-ui/react-switch": "^1.2.2",
-    "@radix-ui/react-tabs": "^1.1.9",
-    "@radix-ui/react-toast": "^1.2.11",
-    "@radix-ui/react-toggle": "^1.1.6",
-    "@radix-ui/react-toggle-group": "^1.1.7",
-    "@radix-ui/react-tooltip": "^1.2.4",
-    "@radix-ui/react-visually-hidden": "^1.2.0",
+    "@phosphor-icons/react": "2.1.7",
+    "@radix-ui/react-accordion": "^1.2.10",
+    "@radix-ui/react-alert-dialog": "^1.1.13",
+    "@radix-ui/react-avatar": "^1.1.9",
+    "@radix-ui/react-checkbox": "^1.3.1",
+    "@radix-ui/react-dialog": "^1.1.13",
+    "@radix-ui/react-dropdown-menu": "^2.1.14",
+    "@radix-ui/react-label": "^2.1.6",
+    "@radix-ui/react-popover": "^1.1.13",
+    "@radix-ui/react-progress": "^1.1.6",
+    "@radix-ui/react-radio-group": "^1.3.6",
+    "@radix-ui/react-scroll-area": "^1.2.8",
+    "@radix-ui/react-select": "^2.2.4",
+    "@radix-ui/react-switch": "^1.2.4",
+    "@radix-ui/react-tabs": "^1.1.11",
+    "@radix-ui/react-toast": "^1.2.13",
+    "@radix-ui/react-toggle": "^1.1.8",
+    "@radix-ui/react-toggle-group": "^1.1.9",
+    "@radix-ui/react-tooltip": "^1.2.6",
+    "@radix-ui/react-visually-hidden": "^1.2.2",
     "@radix-ui/themes": "^3.2.1",
-    "@react-spring/web": "^9.7.5",
-    "@recogito/annotorious-supabase": "^1.3.2",
-    "@recogito/pdf-annotator": "^1.0.2",
-    "@recogito/react-pdf-annotator": "^1.0.2",
-    "@recogito/react-text-annotator": "^3.0.2",
-    "@recogito/studio-sdk": "^0.5.1",
-    "@recogito/text-annotator": "^3.0.2",
-    "@recogito/text-annotator-tei": "^3.0.2",
+    "@react-spring/web": "^10.0.0",
```

Comment:
This was the last dependency that kept us from switching to React 19. I updated this, but kept React at v18. React 19 is something we should test more thoroughly first...